### PR TITLE
Add support for the WNK-MOT1 Wink Motion Sensor

### PR DESCRIPTION
**Pull request by grizzlyjere**

### CODE REVIEW

File: config/leviton/vrpa1.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<Product Revision="1" xmlns="https://github.com/OpenZWave/open-zwave">
+  <MetaData>
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/001D:0334:1805</MetaDataItem>
+    <MetaDataItem name="ProductPic">images/leviton/dz15s.png</MetaDataItem> <!-- This item looks similar to the DZ15s-->
```
Comment (Fishwaldo) [inline config/leviton/vrpa1.xml:4]: Please add a copy - If someone changes dz15s then this device would be affected.

File: config/manufacturer_specific.xml [hunk 2]
```diff
@@ -1,1926 +1,1929 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (Fishwaldo) [inline config/manufacturer_specific.xml:1]: Why the huge change. Please fix up your editor from reformating the entire file and only add the entry that has been added/changed.

**Comment by grizzlyjere** [comment]
> You have two files here. a Leviton/vrpa1 and wink. Did you mean to add both? (thats fine, just confirming)

My apologies, messed up my branching.  Will submit under a separate PR with the changes requested.

File: config/manufacturer_specific.xml (hunk 2 above)
Comment (grizzlyjere) [reply config/manufacturer_specific.xml:1]: Sorry, should be fixed now.  Wasn't sure on file revision to use since it was just updated.

**Comment by Fishwaldo** [comment]
Thanks!

**Pull request merged**